src/app/services/user-service.service.ts

```diff
@@ -24,10 +25,21 @@ export class UserService {
     return this.http.delete('/accounts/user', { responseType: 'text' });
   }
 
-/*  /!**
-   * Adds order to list of orders made by this user.
-   *!/
-  public updateOrders (order: Order){
+  updateUser (user: User){
+    this.userDetails.next (user);
+  }
+
```

Comment:
Is a user subject necessary? There aren't many components that need this data. Did the input bindings not work?